packages/bpk-component-chip-group/src/BpkChipGroup.module.scss

```diff
@@ -42,6 +42,10 @@
   &--wrap {
     flex-wrap: wrap;
   }
+
+  &--withLabel {
```

Comment:
I think this should be `--with-label` according to BEM naming ([docs](https://github.com/Skyscanner/backpack/blob/0d75ec32c3b0fafce594f898f1bcad0b7c4d73b0/CONTRIBUTING.md#css))